README.md

```diff
@@ -177,7 +178,7 @@ def advanced_example_using_database():
         format_list=["pdf", "rtf", "xml"],
         parameters={'python_version': python_version()},
         db_connection=con,
-        locale='pt_BR'  # LOCALE Ex.:(en_US, de_GE)
+        locale='pt_BR'  # LOCALE Ex.:(en, de)
```

Comment:
Not applicable.
The correct is is in the following en_US format, de_GE, pt_BR ...